R/constants.R

```diff
@@ -22,6 +22,8 @@ FLAG_XML <- "FLAG_XML"
 HISTORICAL_YEARS <- 1971:2010
 IMF_GDP_YEARS <- 2010:2020
 FUTURE_YEARS <- seq(2015, 2100, 5)
+BASE_YEARS <- c(1975, 1990, 2005, 2010)
```

Comment:
Should this be `water.BASE_YEARS`? Is this used anywhere else? Same question applies to `MODEL_YEARS` below.